tools/webpack.config.js

```diff
@@ -87,6 +87,11 @@ module.exports = {
           },
         ],
       },
+      {
+        test: /\.tsx?$/,
+        use: 'ts-loader',
```

Comment:
Here we only need the `babel-loader`. You can see here an example 

https://medium.com/@francesco.agnoletto/how-to-set-up-typescript-with-babel-and-webpack-6fba1b6e72d5

Actually, define `babel-loader` is enough.